packages/terra-form-select/src/tag/Frame.jsx

```diff
@@ -642,6 +643,8 @@ class Frame extends React.Component {
     const descriptionId = `terra-select-screen-reader-description-${uniqueid()}`;
     const customAriaDescribedbyIds = customProps['aria-describedby'];
     const ariaDescribedBy = customAriaDescribedbyIds ? `${descriptionId} ${customAriaDescribedbyIds}` : descriptionId;
+    const { id } = customProps;
+    delete customProps.id;
```

Comment:
This can be deconstructed in the props spread above so it doesn't need to be deleted from the customProps. You'll have to add id as a PropType for the linter, but I think that's reasonable since this component does use the id passed in.

```
const propTypes = {
  id: PropTypes.string, 
}
```

    const {
      id,
      ...customProps
    } = this.props;

Reply:
Added id as prop in 5bb136d